src/localization/es/lessons/regexForSeo.json

```diff
@@ -1,49 +1,48 @@
+  "lesson.regexForSeo.maxLimitation.title": "Restricciones de Longitud (Máximo) `{n,m}`",
+  "lesson.regexForSeo.maxLimitation.description": "Para filtrar las URL con una longitud maxima de `35` caracteres, escriba entre llaves los valores numéricos mínimo y máximo que usaremos para la longitud `{1,35}` al lado del punto `.`.",
 
-  "lesson.regexForSeo.maxLimitation.title": "Length Restrictions (Maximum) `{n,m}`",
-  "lesson.regexForSeo.maxLimitation.description": "To filter URLs with a maximum length of `35` characters, type `{1,35}` the minimum and maximum number values we will use for the length, using curly braces at the end of the period `.`.",
+  "lesson.regexForSeo.minLimitation.title": "Restricciones de Longitud (Mínimo) `{n,}`",
+  "lesson.regexForSeo.minLimitation.description": "Para filtrar las URL con una longitud mínima de `35` caracteres, escriba entre llaves y al lado del punto `{35,}` el valor numérico mínimo que usaremos para la longitud con una coma.",
 
-  "lesson.regexForSeo.minLimitation.title": "Length Restrictions (Minumum) `{n,}`",
-  "lesson.regexForSeo.minLimitation.description": "To filter URLs with a minimum length of `35` characters, type `{35,}` using curly braces at the end of the period and ending with a comma for the minimum number value we will use for the length.",
-
-  "lesson.regexForSeo.robots.title": "robots.txt (regex-like syntax)",
-  "lesson.regexForSeo.robots.description": "Here is an example of regex-like usage in the `robots.txt` file. The example states that bots should not crawl links with the extension `pdf`.",
+  "lesson.regexForSeo.robots.title": "robots.txt (sintaxis similar a regex)",
+  "lesson.regexForSeo.robots.description": "Aquí hay un ejemplo simple del uso de regex en un archivo `robots.txt`. El ejemplo establece que los robots no pueden rastrear enlaces con la extensión `pdf`.",
 
   "lesson.regexForSeo.htaccess.title": ".htaccess",
-  "lesson.regexForSeo.htaccess.description": "Here is an example of using regex in the `.htaccess` file. In the example, `php` extension links are directed to `html` extensions with `301` code.",
+  "lesson.regexForSeo.htaccess.description": "Aquí hay un ejemplo de uso de regex en el archivo `.htaccess`. En el ejemplo los enlaces con extensión `php` son dirigidos a extensiones `html` con código `301`.",
 
-  "lesson.regexForSeo.outro.title": "Outro",
-  "lesson.regexForSeo.outro.description": "Regex knowledge is a skill that increases productivity on any platform that allows you to use regex. Once you start using it and improve this skill, it will help you complete many repetitive tasks faster and save time for other things. To better understand the basics of Regex, you can visit the `Regex 101` tutorial. You can use the `Playground` page to test your expressions on your texts."
+  "lesson.regexForSeo.outro.title": "Otro",
```

Comment:
"otro" is used in the `other` sense. But the meaning here is more like the `ending`, the `finale`.

Reply:
I update this last title for `Final`